packages/minimongo/call_in_observe_test.js

```diff
@@ -0,0 +1,34 @@
+// This file is a regression test for https://github.com/meteor/meteor/issues/907
+
+Meteor.methods({
+  isRunningOnServer: function () {
+    return Meteor.isServer;
+  },
+  insertIntoLocalCollection: function () {
+    if (Meteor.isClient) {
+      LocalCollection.insert({});
+    }
+  }
+});
+
+if (Meteor.isClient) {
+  var LocalCollection = new Meteor.Collection(null);
+
+  testAsyncMulti("Meteor.call inside observe sends method to server", [
+    function (test, expect) {
+      var done = expect();
+
+      LocalCollection.find().observe({
+        added: function () {
+          Meteor.call("isRunningOnServer", function (err, res) {
+            test.equal(err, undefined);
```

Comment:
we usually use `test.isFalse(err)`.  (it's kinda weird that we use undefined instead of null here)